src/renderers/dom/client/ReactReconcileTransaction.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactReconcileTransaction
+ * @flow
```

Comment:
It's indeed more interesting to roll files with `@flow` right away. Never used `weak` that much, if the flow team deprecates it, I will stop recommending it then!